hkmc2/shared/src/test/mlscript/codegen/Hygiene.mls

```diff
@@ -146,10 +146,10 @@ cls.bar
 
 fun foo() =
   if false then
-    module A
```

Comment:
I think the point of this test was to use a nested module. Can we just change the definition to `fun foo(): module M` for some `M`?